test/infra/fixtures.py

```diff
@@ -306,16 +305,17 @@ def pytest_collection_modifyitems(config, items):
 
 
 @pytest.fixture(scope="class")
-def service_identifier(service_url: str) -> str:
+def service_issuer(service_url: str) -> str:
     """
-    Get the long term service identifier, under the form of a DID.
+    Get the long term service issuer string.
 
     The service is configured to include this identifier in receipts.
     """
-
     result = urlparse(service_url)
     assert result.hostname is not None
-    return format_did_web(result.hostname, result.port)
+    if result.port is None or result.port == 443 or result.port == 80:
+        return "https://" + result.hostname
```

Comment:
if result.port == 80 it's arguable that this should return "http://" + result.hostname instead.

Reply:
this block was removed